techniques/html/H65.html

```diff
@@ -12,118 +11,127 @@
 type="text" size="3" value="" &gt;
 &lt;input id="lastDigits" name="lastDigits" title="Last four digits of phone number" 
 type="text" size="4" value="" &gt;
-&lt;/fieldset&gt; </pre></section>
+&lt;/fieldset&gt; </pre>
+            <div class="note">
+               <div role="heading" class="note-title marker" aria-level="3">訳注</div>
+               <div>   
+                  <p>WAIC では H65-2 に関する<a href="https://waic.jp/docs/as/">アクセシビリティ サポーテッド（AS）情報</a>を提供している。2014 年 6 月版の<a href="https://waic.jp/docs/as/info/201406/H65-2.html">アクセシビリティ サポーテッド（AS）情報: H65-2</a> では、「要注意」と評価されている。WAIC はウェブ制作者にこの達成方法が一部の環境では動作しないことに注意を促すものである。
+                  </p>  
+               </div>
+            </div>
+            </section>
             <section class="example" id="example-3--a-search-function">
-               <h3>Example 3:  A Search Function</h3>
-               <p>A Web page contains a text field where the user can enter search terms and a button
-                  labeled "Search" for performing the search. The <code class="att">title</code> attribute is used to identify the form control and the button is positioned right
-                  after the text field so that it is clear to the user that the text field is where
-                  the search term should be entered.
+               <h3>例 3:  検索機能</h3>
+               <p>ウェブページに、利用者が検索語を入力できるテキストフィールドと、検索を実行するための「検索」というラベルが付いたボタンがある。フォームコントロールを特定するの <code class="att">title</code> 属性を利用し、ボタンはテキストフィールドの直後に置いてあるので、テキストフィールドに検索語を入力すべきことは利用者にとって明白である。
                </p><pre xml:space="preserve">
 &lt;input type="text" title="Type search term here"/&gt; &lt;input type="submit" value="Search"/&gt;
-</pre></section>
+</pre>
+            <div class="note">
+               <div role="heading" class="note-title marker" aria-level="3">訳注</div>
+               <div>   
+                  <p>WAIC では H65-3 に関する<a href="https://waic.jp/docs/as/">アクセシビリティ サポーテッド（AS）情報</a>を提供している。2014 年 6 月版の<a href="https://waic.jp/docs/as/info/201406/H65-3.html">アクセシビリティ サポーテッド（AS）情報: H65-3</a> では、「要注意」と評価されている。WAIC はウェブ制作者にこの達成方法が一部の環境では動作しないことに注意を促すものである。
+                  </p>  
+               </div>
+            </div>
+            </section>
             <section class="example" id="example-4-a-data-table-of-form-controls">
-               <h3>Example 4: A data table of form controls</h3>
-               <p>A data table of form controls needs to associate each control with the column and
-                  row headers for that cell. Without a title (or off-screen label) it is difficult for
-                  non-visual users to pause and interrogate for corresponding row or column header values
-                  using their assistive technology while tabbing through the form.
+               <h3>例 4: フォームコントロールのデータテーブル</h3>
+               <p>フォームコントロールのデータテーブルでは、各コントロールとそのセルの列及び行ヘッダーを関連付ける必要がある。 title (又はオフスクリーンの label) がないと、視覚を持たない利用者にとっては、フォームをタブで移動しながら支援技術を用いて対応する行あるいは列のヘッダー値を一時停止して調べることが困難である。
```

Comment:
off-screenをそのまま訳して画面外とするのは厳しいので

```suggestion
               <p>フォームコントロールのデータテーブルでは、各コントロールとそのセルの列及び行ヘッダーを関連付ける必要がある。 title (又は画面に現れない label) がないと、視覚を持たない利用者にとっては、フォームをタブで移動しながら支援技術を用いて対応する行又は列のヘッダー値を一時停止して調べることが困難である。
```